WISE-Community/WISE
# Issue 1523 contact wise page

The contact form page can be accessed at this path
http://localhost:8080/wise/site/contact

Not signed in (without recaptcha key set on server)
- Test the form when not signed in

Not signed in (with recaptcha key set on server, you can get a key here https://www.google.com/recaptcha/admin)
- Test the form when not signed in (it should prompt for a recaptcha)

Signed in as teacher (it should not ask for a recaptcha if the user is signed in)
- Open the form from the home page and make sure it works
- Open the form from a run's "Report Problem" option (the email should contain information about the run and project)

Signed in as student (it should not ask for a recaptcha if the user is signed in)
- Open the form from the home page and make sure it works
- Open the form from a run's "Report Problem" (the email should contain information about the run and project)

Closes #1523
